cfgov/paying_for_college/jinja2/paying-for-college/college-cost-blocks/07-other-resources.html

```diff
@@ -86,15 +86,15 @@ <h2>Other sources</h2>
         true,
         'Money saving tips',
         '<ul>
-            <li>Some states offer grants and funding for specific costs related to education, like childcare. Ask the financial aid office or your state or territory\'s child care fund.</li> 
+            <li>Some states offer grants and funding for specific costs related to education, like child care. Ask the financial aid office or <a href="https://childcare.gov/state-resources-home/" class="external-link">your state or territory\'s child care fund</a>.</li> 
```

Comment:
```suggestion
            <li>Some states offer grants and funding for specific costs related to education, like child care. Ask the financial aid office or <a href="https://childcare.gov/state-resources-home/" target="_blank" rel="noreferrer noopener">your state or territory\'s child care fund</a>.</li> 
```
We probably want the link to open in a new window since that's the pattern we've been using elsewhere in the app. I also think the `external-link` class is unnecessary -- the link middleware will handle adding the icon.